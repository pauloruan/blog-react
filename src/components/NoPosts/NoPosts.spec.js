import { render, screen } from '@testing-library/react';
import { NoPosts } from '.';

describe('<NoPosts />', () => {
  it('should render the text', () => {
    expect(screen.queryByText(/nÃ£o existem posts/i)).not.toBeInTheDocument();
  });

  it('should match snapshot', () => {
    const { container } = render(<NoPosts />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
