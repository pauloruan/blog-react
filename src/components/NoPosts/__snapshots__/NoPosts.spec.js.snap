// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoPosts /> should match snapshot 1`] = `
<div
  class="no-posts"
>
  <p>
    NÃ£o existem posts.
  </p>
</div>
`;
